src/Debugger.ts

```diff
@@ -0,0 +1,25 @@
+export class Debugger {
```

Comment:
for consistencies sake, can you use a lowercase filename please?

Reply:
sure :)